WordPress/src/main/java/org/wordpress/android/ui/sitecreation/domains/SiteCreationDomainsFragment.kt

```diff
@@ -88,20 +77,17 @@ class SiteCreationDomainsFragment : SiteCreationBaseFormFragment() {
         super.onViewStateRestored(savedInstanceState)
         // we need to set the `onTextChanged` after the viewState has been restored otherwise the viewModel.updateQuery
         // is called when the system sets the restored value to the EditText which results in an unnecessary request
-        searchInputWithHeader.onTextChanged = { viewModel.updateQuery(it) }
+        searchInputWithHeader?.onTextChanged = { viewModel.updateQuery(it) }
     }
 
     private fun initRecyclerView(rootView: ViewGroup) {
```

Comment:
It looks like we don't need this parameter anymore.

```suggestion
    private fun initRecyclerView() {
```